backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -48,6 +52,62 @@
 class PromptStudioHelper:
     """Helper class for Custom tool operations."""
 
+    @staticmethod
+    def create_default_profile_manager(user: User, tool_id: uuid) -> None:
+        """Create a default profile manager for a given user and tool.
+
+        Args:
+            user (User): The user for whom the default profile manager is
+            created.
+            tool_id (uuid): The ID of the tool for which the default profile
+            manager is created.
+
+        Raises:
+            AdapterInstance.DoesNotExist: If no suitable adapter instance is
+            found for creating the default profile manager.
+
+        Returns:
+            None
+        """
+        try:
+            AdapterInstance.objects.get(
+                is_friction_less=True,
+                is_usable=True,
+                adapter_type=AdapterKeys.LLM,
+            )
+
+            default_adapters: BaseManager[AdapterInstance] = (
+                AdapterInstance.objects.filter(is_friction_less=True)
+            )
+
+            profile_manager = ProfileManager(
+                prompt_studio_tool=CustomTool.objects.get(pk=tool_id),
+                is_default=True,
+                created_by=user,
+                modified_by=user,
+                chunk_size=0,
```

Comment:
Is setting chunk size to 0 a good idea? If they upload large documents it would fail - this would depend on the LLM's context window size. However setting it to 1024 will allow it to atleast run for all cases even if its a bit slow
CC: @johnyrahul @arun-venkataswamy

Reply:
@chandrasekharan-zipstack  this is as per naren's request. your thoughts on this @shuveb  and @arun-venkataswamy ??

Reply:
As discussed will go  with zero chunk size and will add a ticket to warn users to optimize chunk  size configuration manually in UI
cc: @chandrasekharan-zipstack @nehabagdia @VikashPratheepan @gopalsamyperumal